src/pybind/mgr/dashboard/run-frontend-i18n.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd frontend
+
+if [ `uname` != "FreeBSD" ]; then
+  . $CEPH_ROOT/build/src/pybind/mgr/dashboard/node-env/bin/activate ||
+  echo 'Info: missing node-env'
+fi
+
+npm run i18n
+
+count=`git status | grep "messages.xlf" -c`
```

Comment:
This looks somewhat brittle. Wouldn't it be better to use git to check the modification status of this file directly, e.g. via `git status -s messages.xlf`